Document that returning `sys.monitoring.DISABLE` in response to a global event raises `ValueError` (#140726)
python/cpython: Doc/library/sys.monitoring.rst

   A special value that can be returned from a callback function to disable
   events for the current code location.

:ref:`Local events <monitoring-event-local>` can be disabled for a specific code
location by returning :data:`sys.monitoring.DISABLE` from a callback function.
This does not change which events are set, or any other code locations for the
same event.

Disabling events for specific locations is very important for high
performance monitoring. For example, a program can be run under a
debugger with no overhead if the debugger disables all monitoring
except for a few breakpoints.

If :data:`DISABLE` is returned by a callback for a
:ref:`global event <monitoring-event-global>`, :exc:`ValueError` will be raised

> by the interpreter in a non-specific location (i.e. no traceback will be

Reviewer: ```suggestion
by the interpreter in a non-specific location (that is, no traceback will be
```
Per the [Docs style guide](https://devguide.python.org/documentation/style-guide/#use-simple-language), latin abbreviations should not be used.
